packages/block-library/src/spacer/edit.js

```diff
@@ -97,88 +143,64 @@ const SpacerEdit = ( {
 						bottomLeft: false,
 						topLeft: false,
 					} }
+					orientation={ blockOrientation }
+					max={ MAX_SPACER_WIDTH }
 					onResizeStart={ handleOnResizeStart }
+					onResize={ setTemporaryWidth }
 					onResizeStop={ handleOnHorizontalResizeStop }
-					showHandle={ isSelected }
-					__experimentalShowTooltip={ true }
-					__experimentalTooltipProps={ {
-						axis: 'x',
-						position: 'corner',
-						isVisible: isResizing,
-					} }
+					isSelected={ isSelected }
 				/>
 			);
 		}
 
 		return (
-			<ResizableBox
-				className={ classnames(
-					'block-library-spacer__resize-container',
-					{
-						'is-selected': isSelected,
-					}
-				) }
-				size={ {
-					height,
-				} }
-				minHeight={ MIN_SPACER_HEIGHT }
-				enable={ {
-					top: false,
-					right: false,
-					bottom: true,
-					left: false,
-					topRight: false,
-					bottomRight: false,
-					bottomLeft: false,
-					topLeft: false,
-				} }
-				onResizeStart={ handleOnResizeStart }
-				onResizeStop={ handleOnVerticalResizeStop }
-				showHandle={ isSelected }
-				__experimentalShowTooltip={ true }
-				__experimentalTooltipProps={ {
-					axis: 'y',
-					position: 'bottom',
-					isVisible: isResizing,
-				} }
-			/>
+			<>
+				<ResizableSpacer
+					enable={ {
+						top: false,
+						right: false,
+						bottom: true,
+						left: false,
+						topRight: false,
+						bottomRight: false,
+						bottomLeft: false,
+						topLeft: false,
+					} }
+					orientation={ blockOrientation }
+					max={ MAX_SPACER_HEIGHT }
+					onResizeStart={ handleOnResizeStart }
+					onResize={ setTemporaryHeight }
+					onResizeStop={ handleOnVerticalResizeStop }
+					isSelected={ isSelected }
+				/>
+			</>
 		);
 	};
 
 	useEffect( () => {
 		if ( orientation === 'horizontal' && ! width ) {
-			updateWidth( 72 );
-			updateHeight( 0 );
+			setAttributes( {
+				height: parseFloat( 0 ),
```

Comment:
Is there a reason for parsing hardcoded primitive values?

Reply:
Definitely not -- fixed.